msteams-platform/messaging-extensions/how-to/authentication/msgex-sso-graph-api1.md

```diff
@@ -0,0 +1,144 @@
+
+    The added permissions are displayed in the **API permissions** page.
+
+   :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/configured-permissions.png" alt-text="API permissions are configured." border="true":::
+
+    You've configured your app with Microsoft Graph permissions.
+
+## Configure authentication for different platforms
+
+Configuration for authentication depends on the platform or device where you want to target your app. You may need to configure redirect URIs, specific authentication settings, or details specific to the platform.
+
+> [!NOTE]
+>
+> - If your bot app hasn't been granted IT admin consent, app users have to provide consent the first time they use your app on a different platform.
+> - Implicit grant is not required if SSO is enabled on a bot app.
+
+You can configure authentication for multiple platforms as long as the URL is unique.
+
+### To configure authentication for a platform
+
+1. Open the app you registered in the [Azure portal](https://ms.portal.azure.com/).
+
+1. Select **Manage** > **Authentication** from the left pane.
+
+    :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/azure-portal-platform.png" alt-text="Authenticate for platforms" border="true":::
+
+    The **Platform configurations** page appears.
+
+1. Select **+ Add a platform**.
+
+    :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/add-platform.png" alt-text="Add a platforms" border="true":::
+
+    The **Configure platforms** page appears.
+
+1. Select the platform that you want to configure for your bot app. You can choose the platform type from web or SPA.
+
+    :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/configure-platform.png" alt-text="Select web platform" border="true":::
+
+    You can configure multiple platforms for a particular platform type. Ensure that the redirect URI is unique for every platform you configure.
+
+    The Configure Web page appears.
+
+    > [!NOTE]
+    > The configurations will be different based on the platform you select.
+
+1. Enter the configuration details for the platform.
+
+    :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/config-web-platform.png" alt-text="Configure web platform" border="true":::
+
+    1. Enter the redirect URI. The URI should be unique.
+    2. Enter the front-channel log-out URL.
+    3. Select the tokens you want Azure AD to send for your app.
+
+1. Select **Configure**.
+
+    The platform is configured and displayed in the **Platform configurations** page.
+
+## Acquire access token for MS Graph
+
+You'll need to acquire access token for Microsoft Graph.
+
+### Configure code to fetch access token
+
+The following code provides an example of using the access token obtained in the OAuth process to submit a request for permissions to Microsoft Graph.
```

Comment:
The request, based on the signature, appears to be fetching the user's email address not for "permissions". 

The access token is granting the bearer (this bot) authorization to read the user's email address and we can attach it to a request asking for the user's email address.